services/rfq/relayer/quoter/quoter.go

```diff
@@ -519,17 +550,32 @@ func (m *Manager) getOriginAmount(parentCtx context.Context, origin, dest int, a
 		quoteAmount = minQuoteAmount
 	}
 
-	// Finally, clip the quoteAmount by the balance
-	if quoteAmount.Cmp(balance) > 0 {
-		span.AddEvent("quote amount greater than balance", trace.WithAttributes(
+	// Clip the quoteAmount by the max origin balance
+	maxBalance := m.config.GetMaxBalance(input.OriginChainID, input.OriginTokenAddr)
+	if input.OriginBalance != nil && maxBalance.Cmp(big.NewInt(0)) > 0 {
```

Comment:
See comment above re: default value for `maxBalance` - this brings the question how do we want to treat `maxBalance == 0` case

Reply:
Current thinking is to use `nil` for 'positive inf'- this way an unspecified `max_balance` param in the config will mean no max balance value, explicitly setting it to zero should result in all quotes to this chain having 0 quoteAmount